packages/lesswrong/lib/vulcan-users/permissions.ts

```diff
@@ -164,6 +164,23 @@ export const userCanComment = (user: PermissionableUser|DbUser|null): boolean =>
   return true;
 }
 
+// Same as userCanComment, but without the unreviewed author check
+export const userCanQuickTake = (user: PermissionableUser|DbUser|null): boolean => {
```

Comment:
🤔 Someone reported an issue where they couldn't see the form despite being reviewed (probably because they hadn't created a quick take before).  I guess if it shows up for logged out users it'll probably show up for them too, but I'm a bit confused by how that happened if the visibility was previously gated by `userCanComment`.

Reply:
>probably because they hadn't created a quicktake before
That doesn't make sense as an explanation given the prior logic, so I'm confused too if their report is accurate

Reply:
There's a callback that creates a quick take container post if needed, and nothing that would block display of the submission box on it not existing